database/index.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+const Transaction = require('./Transaction.js');
+const Stock = require('./Stock.js');
+
+// connect to mongo via localhost
+var promise = mongoose.connect('mongodb://localhost/robinsHood', {
```

Comment:
I'd probably call this `var connection` over `var promise`

Reply:
changed